hardhat.config.ts

```diff
@@ -88,6 +88,7 @@ const config: HardhatUserConfig = {
 task('setStages', 'Set stages for ERC721M')
   .addParam('contract', 'contract address')
   .addParam('stages', 'stages json file')
+  .addOptionalParam('gasprice', 'Set gas price in Gwei')
```

Comment:
can you rename this as `gaspricegwei`? Because in this context, price are in wei by default.